analyzer/src/main/kotlin/managers/Pub.kt

```diff
@@ -71,7 +71,7 @@ private const val PUB_LOCK_FILE = "pubspec.lock"
 private val flutterCommand = if (Os.isWindows) "flutter.bat" else "flutter"
 private val dartCommand = if (Os.isWindows) "dart.bat" else "dart"
 
-private val flutterVersion = Os.env["FLUTTER_VERSION"] ?: "3.0.2-stable"
+private val flutterVersion = Os.env["FLUTTER_VERSION"] ?: "3.0.3-stable"
```

Comment:
Commit message: Please mention that release notes are not yet published, but might become available here: https://github.com/flutter/flutter/wiki/Hotfixes-to-the-Stable-Channel#flutter-30-changes